wpiutil/src/test/native/cpp/test_optional.cpp

```diff
@@ -622,8 +622,8 @@ TEST(Optional, example_rationale)
   /////////////////////////////////
   {
   typedef int T;
-  optional<optional<T>> ot {in_place};
-  optional<optional<T>> ou {in_place, nullopt};
+  //optional<optional<T>> ot {in_place};
+  //optional<optional<T>> ou {in_place, nullopt};
```

Comment:
What’s the error you are getting?  I would prefer to actually fix the implementation rather than disabling the test

Reply:
Unused variable.  I can remove the lines instead

Reply:
oh.  just stick a `(void) ot; (void) ou;` somewhere later in the function then.

Reply:
ok